test/functional/specs/Personalization/C8631576.js

```diff
@@ -0,0 +1,58 @@
+import { t } from "testcafe";
+import createNetworkLogger from "../../helpers/networkLogger";
+import { responseStatus } from "../../helpers/assertions/index";
+import createFixture from "../../helpers/createFixture";
+import {
+  compose,
+  orgMainConfigMain,
+  debugEnabled
+} from "../../helpers/constants/configParts";
+import createAlloyProxy from "../../helpers/createAlloyProxy";
+import isUserAgentClientHintsSupported from "../../helpers/isUserAgentClientHintsSupported";
+
+const networkLogger = createNetworkLogger();
+const config = compose(orgMainConfigMain, debugEnabled);
+
+const ID = "C8631576";
+const DESCRIPTION = `${ID} - Visitor should qualify for an experience based on low entropy client hint`;
+
+createFixture({
+  title: DESCRIPTION,
+  requestHooks: [networkLogger.edgeEndpointLogs]
+});
+
+test.meta({
+  ID,
+  SEVERITY: "P0",
+  TEST_RUN: "Regression"
+});
+
+const sendEventOptions = {
+  decisionScopes: ["chromeBrowserClientHint"]
+};
+
+test(DESCRIPTION, async () => {
+  const alloy = createAlloyProxy();
+  await alloy.configure(config);
+  const eventResult = await alloy.sendEvent(sendEventOptions);
+
+  await responseStatus(networkLogger.edgeEndpointLogs.requests, 200);
+  await t.expect(networkLogger.edgeEndpointLogs.requests.length).eql(1);
+
+  // Tests must be run using https otherwise this will return false
+  if (await isUserAgentClientHintsSupported()) {
```

Comment:
I'm always a little wary of conditionals in tests, so I just want to confirm the behavior: if an end user client device does not support user agent client hints, then they should not qualify for the experiences, and this test is also testing that outcome. "
Does that sound correct?

Reply:
@carterworks, that's a great point - I will add some assertions for the case when client hints are not supported (i.e. user is on Firefox or Safari).  You're correct that in that case the user should not qualify for the experience.

Reply:
Nicely done.